### PR TITLE
AN-2452 timzones support

### PR DESCRIPTION
**Pull request by sabe25**
How to test
- ping me for report 
- you will see two charts but the data is kinda shifted by one day
- click on one chart and edit the datasource
- set the timezone to `+02:00` (same as license) and save
- the two charts` data should align again

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -450,6 +464,32 @@ function validateConfig(configParams) {
   }
 }
 
+/**
+ * converts a timezone string into the minute repräsentation of it
```
Comment (paulk4633) [inline src/main.js]:
```suggestion
 * converts a timezone string into the minute representation of it
```

**Review by paulk4633 — APPROVED** [review]
Nice 👍

File: src/main.js [hunk 2]
```diff
@@ -450,6 +464,32 @@ function validateConfig(configParams) {
...
+ * @param {string} timezone 
+ * @returns {number | null} timezone in minutes or null if invalid
+ */
+ function convertTimezoneToMinutesOffset(timezone) {
```
Comment (peterpf) [inline src/main.js:472]: Check out https://github.com/bitmovin-engineering/dashboard/blob/develop/src/utils/generic/DateUtils.ts#L217.
This is how we do it in the dashboard and covers more cases.

**Review by peterpf — APPROVED** [review]
Looks good from my side. Added a comment on how we do the timezoneOffset extraction in the dashboard.

File: src/main.js (hunk 2 above)
Comment (sabe25) [reply src/main.js:472]: Thanks for the reference i tried this code but beside the issues i had with converting this to plain javascript it didn't worked out as well as the code before

**Pull request merged**